tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/client/ConsumerTest.java

```diff
@@ -446,7 +449,15 @@ public void internalSimpleSend(int protocolSender, int protocolConsumer) throws
          TextMessage msg = session.createTextMessage("hello");
          msg.setIntProperty("mycount", 0);
          producer.send(msg);
-         connection.close();
+
+         StringBuffer bufferLarge = new StringBuffer();
```

Comment:
Would be better to use `StringBuilder` instead, because `StringBuffer` is `synchronized`